src/chocolatey/infrastructure.app/nuget/NugetCommon.cs

```diff
@@ -160,13 +200,30 @@ public static IPackageRepository GetRemoteRepository(ChocolateyConfiguration con
                 configuration.Sources = updatedSources.Remove(updatedSources.Length - 1, 1).to_string();
             }
 
-            var repository = new AggregateRepository(repositories, ignoreFailingRepositories: true)
+            if (priorityRepositories.Count == 1 && repositories.Count == 0)
+            {
+                repositories = priorityRepositories[0];

```

Comment:
Oversight, need to clear out the dictionary here afterwards, so we don't re-add the sources in the foreach loop.